docs/fermi_hubbard/index.md

```diff
@@ -0,0 +1,23 @@
+# Fermi-Hubbard
+
+Strongly correlated quantum systems give rise to many exotic physical phenomena, 
+including high-temperature superconductivity. 
+Simulating these systems on quantum computers may avoid the prohibitively 
+high computational cost incurred in classical approaches. 
+However, systematic errors and decoherence effects presented in current quantum 
+devices make it difficult to achieve this. 
+We simulated the dynamics of the one-dimensional Fermi-Hubbard model 
```

Comment:
Generally avoid "we" in developer docs since it's less clear who authored the doc. Can write around it ("In this experiment the ...") or be explicit ("The Google quantum team ...")